src/algorithms/rhythm/rhythmtransform.cpp

```diff
@@ -49,63 +49,83 @@ void RhythmTransform::compute() {
   vector<vector<Real> >& output = _rhythmTransform.get();
 
   int nFrames = bands.size();
-  int nBands = bands[0].size();
+  // 
+  // Check first that we have a valid populated input to avoid  core dump
+  if (nFrames== 0) {
+    throw EssentiaException("File looks like a completely silent file");
```

Comment:
Use more informative messages for exceptions: 
- Mention the algorithm name (`RhythmTransform: Exception message`) similar other algorithms.
- RhythmTransform does not work with files, therefore this message is misleading. We can say "Input mel-spectrogram is empty".